specification/ai/Azure.AI.Projects/evaluations/routes.tsp

```diff
@@ -14,21 +16,39 @@ alias ServiceTraits = SupportsClientRequestId &
   NoRepeatableRequests &
   NoConditionalRequests;
 
-alias EvaluationsOperations = Azure.Core.ResourceOperations<ServiceTraits>;
+alias RetryServiceTrait = {
+  @header("Retry-After")
+  @doc("Recommended time interval in seconds before making another request when the evaluation is still processing. This helps prevent excessive polling and reduces server load during long-running evaluations.")
+  @added(Versions.v2025_07_31_preview)
+  @visibility(Lifecycle.Read)
+  retryAfter?: string;
+};
+
+alias DisplayPreferencesTypeParameterTrait = {
+  @doc("Filter expression to narrow down the list of evaluations based on specific criteria. Supports filtering by properties such as status, tags, or properties to help manage large evaluation collections.")
+  @query
+  @added(Versions.v2025_07_31_preview)
+  filter?: string;
+};
+
+alias EvaluationsOperations = ResourceOperations<ServiceTraits>;
 
 @route("evaluations")
 @added(Versions.v2025_05_15_preview)
-@removed(Versions.v1)
 interface Evaluations {
   @doc("Get an evaluation run by name.")
-  get is EvaluationsOperations.ResourceRead<Evaluation>;
+  get is EvaluationsOperations.ResourceRead<Evaluation & RetryServiceTrait>;
 
   @doc("List evaluation runs")
-  list is EvaluationsOperations.ResourceList<Evaluation>;
+  list is EvaluationsOperations.ResourceList<
+    Evaluation,
+    ListQueryParametersTrait<DisplayPreferencesTypeParameterTrait>
+  >;
 
   #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
   @doc("Creates an evaluation run.")
   @route("runs:run")
+  @removed(Versions.v2025_07_31_preview)
```

Comment:
Let's keep the existing API, otherwise when the new SDK generated, the APIs will disappeared immediately. We can mark it as deprecated: https://typespec.io/docs/language-basics/directives/